Riot/ViewController/MasterTabBarController.m

```diff
@@ -77,6 +77,15 @@ - (void)viewDidLoad
     _roomsViewController = [self.viewControllers objectAtIndex:TABBAR_ROOMS_INDEX];
     _groupsViewController = [self.viewControllers objectAtIndex:TABBAR_GROUPS_INDEX];
     
+    // Set the accessibility labels for all buttons #1842
+    [_settingsBarButtonItem setAccessibilityLabel:@"Settings"];
```

Comment:
These strings need to be internationalised using `NSLocalizedStringFromTable(@"xxx", @"Vector", nil);`

Default English values should be then added the `en.lproj/Vector.string` file.

Reply:
I'll try to add this.